File: JAVA PROGRAMMING/Test52.java
// 52. TERNARY OPERATOR
import java.util.Scanner;
public class Test {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);

        System.out.println("Enter Num1 Value: ");
        int num1 = in.nextInt();

        System.out.println("Enter Num2 Value: ");
        int num2 = in.nextInt();

        String res = (num1 > num2) ? "Num1 Greater Than Num2" : "Num2 Greater Than Num1";
        System.out.println(res);

        in.close();
    }
}

File: TYPESCRIPT PROGRAMMING/Test52.ts
// 52. TERNARY OPERATOR
const TernaryOperator = (ip1: number, ip2: number): String => {
    let res: String = (ip1 > ip2) ? `${ip1} Grater Than ${ip2}` : `${ip2} Greater Than ${ip1}`;
    return res;
}
TernaryOperator(10, 20);
TernaryOperator(30, 20);